src/app/page.vue

```diff
@@ -0,0 +1,253 @@
+
+        	// Assign title
+        	this.head.titleTemplate = `${prismicMetaObject.title} ${prismicMetaObject.title_separator} %s`;
+        	// Store the home's title as the default title
+        	this.head.titleDefault = prismicMetaObject.title_home;
+        	// If a pageTitle hasn't been assigned yet, use the default
+        	if( !this.head.title ) this.head.title = prismicMetaObject.title_home;
+
+        	// Assign meta tags
+        	this.head.meta.push( { "property" : "og:title", 		"content" : prismicMetaObject.title } );
+        	this.head.meta.push( { "property" : "og:site_name", 	"content" : prismicMetaObject.site_name } );
+        	this.head.meta.push( { "property" : "description", 		"content" : prismicMetaObject.description } );
+        	this.head.meta.push( { "property" : "og:description",	"content" : prismicMetaObject.description } );
+        	this.head.meta.push( { "property" : "og:url",			"content" : prismicMetaObject.url } );
+        	this.head.meta.push( { "property" : "og:url",			"content" : prismicMetaObject.url } );
+        	this.head.meta.push( { "property" : "og:image",			"content" : prismicMetaObject.image.url } );
+
+        	// Assign favicon
+        	this.head.link.push( { "rel": 'icon', 					"href" : prismicMetaObject.favicon.url, "type" : "image/png" } );
+        	this.head.link.push( { "rel": 'shortcut icon', 			"href" : prismicMetaObject.favicon.url, "type" : "image/png" } );
+        },
+
+		showLoading( isHome )
+		{
+			// If home, only show logo. If not, only show title
+			this.$refs.loading_logo.style.display = (isHome) ? 'block' : 'none';
+			this.$refs.loading_name.style.display = (isHome) ? 'none' : 'block';
+
+			// Show the loading screen
+			this.$refs.loading.classList.add("-shown");
+
+			// Hide after delay
+			this.loadingInt = window.setTimeout( this.hideLoading, 2000 );
+		},
+
+		hideLoading ()
+		{
+			// Hide the loading screen
+			this.$refs.loading.classList.remove("-shown");
+		},
+	},
+
+	"computed" : 
+	{
+		pageTitle() { return this.$store.state.pageTitle },
+	},
+}
+</script>
+
+***********************************************************
+
+<style lang="scss" scoped>
+@import "~styles/_vars.scss";
+
+///////////////////////////////////////////////////////////
+// 	...
+///////////////////////////////////////////////////////////
+
+@keyframes colorBlend 
+{
+	0% 		{ background-color: #143e3b; }
+	14% 	{ background-color: #d6a49a; }
+	28% 	{ background-color: #dddfe2; }
+	42% 	{ background-color: #282626; }
+	57% 	{ background-color: #fbaf23; }
+	71% 	{ background-color: #bd401b; }
+	100% 	{ background-color: #143e3b; }
+}
+
+.loading-screen 
+{
+	visibility: hidden;
+	opacity: 0;
+
+	position: fixed;
+	z-index: 200;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
+
+	animation: colorBlend 6s infinite;
+	animation-direction: alternate;
+	
+	$time : 1.5s;
+	transition: opacity $time,
+				visibility $time;
+
+	cursor: none;
+	font-family: GTWalsheim-Light, sans-serif;
```

Comment:
you can remove this `font-family` property and add `font-weight: 300` because `font-family` will be inherited from the font set on `body`.

Reply:
Got it, fixed with 1260876